test/test_connection.py

```diff
@@ -269,6 +269,10 @@ def test_docs(conn):
     docs.clear()
     assert docs.size() == 0
 
+def test_unicode(conn):
+    conn.begin()
+    conn.add(content.Raw(':βüãäoñr̈ a :Employee .'.encode('utf-8'), content_types.TURTLE))
```

Comment:
I'm curious if setting `content_encoding='utf-8'` also works, I see that if you supply that argument it does get set in the headers of the api call. It seems what this is doing is actually saying that `content.Raw` must take a byte stream to work properly, it would be good to figure out where exactly the issue is with unicode rather than pystardog vs Stardog.

Reply:
doing the following:
>     conn.add(content.Raw(':βüãäoñr̈ a :Employee .', content_encoding='utf-8'))
tries to send the following requests
> {
	'params': {
		'graph-uri': None
	},
	'headers': {
		'Content-Type': None,
		'Content-Encoding': 'utf-8'
	},
	'data': ':βüãäoñr̈ a :Employee .'
}
but it errors with:
```
UnicodeEncodeError: 'latin-1' codec can't encode character '\u03b2' in position 1: Body ('β') is not valid Latin-1. Use body.encode('utf-8') if you want to send it encoded in UTF-8.
```

whereas the working version sends the data already encoded:
```
conn.add(content.Raw(':βüãäoñr̈ a :Employee .'.encode('utf-8'), content_types.TURTLE))
```
data sent:
> {
	'params': {
		'graph-uri': None
	},
	'headers': {
		'Content-Type': 'text/turtle',
		'Content-Encoding': None
	},
	'data': b ':\xce\xb2\xc3\xbc\xc3\xa3\xc3\xa4o\xc3\xb1r\xcc\x88 a :Employee .'
}

I think `content_encoding='utf-8'` doesn't work because the requests session has problem handling utf-8 strings, regardless of the headers set.